client/src/common/components/Cards/Card.tsx

```diff
@@ -71,7 +71,7 @@ export const Card: React.FC<CardProps> = ({
   children,
 }) => (
   <Shadow>
-    <Wrapper onPress={onPress}>
+    <Wrapper onPress={onPress} activeOpacity={0.7}>
```

Comment:
Nitpick… For the future I suspect that we will miss adding this prop at a lot of places. Probably not a biggie. But maybe good to find a global/reusable solution to this? Like we had in the old app, a reusable component. Or some other way.

Reply:
Yeah! Crossed my mind, thanks for pointing it out! Love nitpicks 😊